.github/workflows/ci_linux_x64-libshortfin.yml

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
I suspect this shebang is breaking syntax highlighting in this file by having GitHub treat the file as a bash script instead of a yaml file.

Reply:
Thanks, accidentally copied that with the header. Will push the fix with one of the next refactorings.